src/analyzer/analyzerbeats.cpp

```diff
@@ -287,7 +286,7 @@ void AnalyzerBeats::storeResults(TrackPointer tio) {
     double currentFirstBeat = pCurrentBeats->findNextBeat(0);
     double newFirstBeat = pBeats->findNextBeat(0);
     if (currentFirstBeat == 0.0 && newFirstBeat > 0) {
-        pCurrentBeats->translate(newFirstBeat);
+        tio->setBeats(pCurrentBeats->translate(newFirstBeat));
```

Comment:
I don't want you to make large renamings, but if I just look at the diff I have no idea what `tio` is.